guesser.rb

```diff
@@ -0,0 +1,42 @@
+playing_game = true
+
+while playing_game
+    high = 100
+    low = 1
+    
+    puts "Think of a number between 1 and 100 and I will try to guess it"
+    puts "Tell me if it's 'too high' 'too low' or 'correct' (in those exact words)"
+
+    correctly_guessed = false
+
+    while !correctly_guessed
+        guess = (high + low) / 2
+        puts "My first guess is #{guess}, is that correct?"
```

Comment:
Nit: I'm not sure it's the first guess after the first iteration?

```suggestion
        puts "My guess is #{guess}, is that correct?"
```